pythonosc/test/test_dispatcher.py

```diff
@@ -121,5 +121,14 @@ def test_multiple_handlers_with_wildcard_map(self):
     self.sortAndAssertSequenceEqual(
         [Handler(1, []), Handler(2, [])], self.dispatcher.handlers_for_address("/foo/bar"))
 
+  def test_unmap(self):
```

Comment:
please add a test that checks for the key presence as I hinted before (you can use an empty dispatcher with no handlers mapped for example)